tests/test_metric_range_df.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import json
+import os
+from prometheus_api_client import MetricRangeDataFrame
+
+
+class TestMetricRangeDataFrame(unittest.TestCase):
+    def setUp(self):
+        """
+        read metrics stored as jsons in './tests/metrics'
+        """
+        self.raw_metrics_list = list()
+        self.raw_metrics_labels = list()
+        for (dir_path, _, file_names) in os.walk("./tests/metrics"):
+            for fname in file_names:
+                with open(os.path.join(dir_path, fname), "rb") as f:
+                    metric_jsons = json.load(f)
+
+                # save json list
+                self.raw_metrics_list.extend([metric_jsons])
+
+                # save label configs
+                labels = set()
+                for i in metric_jsons:
+                    labels.update(set(i["metric"].keys()))
+                self.raw_metrics_labels.append(labels)
+
+    def test_setup(self):
+        """
+        Check if setup was done correctly
+        """
+        self.assertEqual(
+            8, len(self.raw_metrics_list), "incorrect number json files read (incorrect test setup)"
+        )
+
+    def test_init_shape(self):
+        """
+        Test if dataframe initialized is of correct shape
+        """
+        # check shape
+        # each metric json contains number of timestamps equal to number entries * number of timestamps in each series 
+        # we're assuming each series has the same number of timestamps
+        # 3 labels
+        for current_metric_list in self.raw_metrics_list:
+            df = MetricRangeDataFrame(current_metric_list)
+            num_values = sum([len(v["values"]) for v in current_metric_list])
+            self.assertEqual(
+                (len(df.index.values), 5),    # shape[1] = 4xlabels + value
+                (num_values, 5),
```

Comment:
```suggestion
                (len(df.index.values), df.shape[1]),    # shape[1] = 4xlabels + value
                (num_values, 5),
```
Would it make sense to use `df.shape[1]` instead of `5`? Because otherwise, unless I misunderstood something, we're just comparing the literal `5` with itself.